Update home-content.md

calderEDJ opened the pull request:
Updated with revised copy. Not sure what the process is for adding the event graphic at the bottom.

calderEDJ committed:
Update home-content.md

Updated with revised copy. Not sure what the process is for adding the event graphic at the bottom.

Frijol commented:
Doc to compare to (for my quick reference): https://docs.google.com/document/d/1Bde1qfxshfwk9y7S2NgJnpFm5VGLYzKuHOqucWjOdEs/edit

Frijol committed:
adds image for release event

Frijol commented on content/home/home-content.md:
```diff
@@ -2,7 +2,10 @@
 <big>Strong environmental regulation is important, but it’s only as good as its enforcement.</big>
 
-EPA is overseen by two congressional committees, the House Energy and Commerce Committee (E&C) and the Senate Environmental and Public Works Committee (EPW). EDGI took a look at environmental enforcement in committee members' home districts. Here's where those reps come from:
+The Environmental Protection Agency (EPA) is charged by Congress to enforce laws that protect people from air pollution, water pollution, and hazardous waste. Two key congressional committees oversee the EPA to ensure it meets its mandate: the <a href="https://energycommerce.house.gov/" target=_blank >House Energy and Commerce Committee</a> and the <a href="https://www.epw.senate.gov/public/" tar ...
```
```suggestion
The Environmental Protection Agency (EPA) is charged by Congress to enforce laws that protect people from air pollution, water pollution, and hazardous waste. Two key congressional committees oversee the EPA to ensure it meets its mandate: the <a href="https://energycommerce.house.gov/" target=_blank >House Energy and Commerce Committee</a> and the <a href="https://www.epw.senate.gov/public/" target=_blank >Senate Environment and Public Works Committee</a>. Based on an analysis of EPA data, EEW Congressional Report Cards give the members of these Congressional committees—and their constituents—the chance to evaluate whether the EPA is fulfilling its mandate in their state/district. Eventually, we hope to compile reports for every district in the U.S.
```

Frijol replied:
U.S. is more correct than US

Frijol commented on content/home/home-content.md:
```diff
@@ -17,6 +20,14 @@ EPA is overseen by two congressional committees, the House Energy and Commerce C
-**Environmental Enforcement Watch** is series of networked online workshops that bring people together from different backgrounds to analyze the data from Environmental Protection Agency’s <a href="https://echo.epa.gov/" target=_blank >Enforcement and Compliance History Online</a> (ECHO) database. ECHO data includes reported violations of environmental laws, facility inspections, and enforcement  ...
+We believe in the public’s right to know, and that we can collaboratively design public open-source systems to analyze, discuss, and act on environmental data collectively. EPA data is available to the public through the <a href="https://echo.epa.gov/" target=_blank >Enforcement and Compliance History Online</a> (ECHO) database. ECHO data includes reported violations of environmental laws such as ...
+
+<big>Upcoming Events:</big>
```
Since this is a heading for a page section, it would be more appropriate to use `#` or `##` instead of `<big>`

Frijol commented on content/home/home-content.md:
```diff
@@ -17,6 +20,14 @@ EPA is overseen by two congressional committees, the House Energy and Commerce C
+We believe in the public’s right to know, and that we can collaboratively design public open-source systems to analyze, discuss, and act on environmental data collectively. EPA data is available to the public through the <a href="https://echo.epa.gov/" target=_blank >Enforcement and Compliance History Online</a> (ECHO) database. ECHO data includes reported violations of environmental laws such as ...
+
+<big>Upcoming Events:</big>
+
```
Consider adding the event details in plaintext here in addition to the graphic, for greater accessibility

Frijol commented on content/home/home-content.md:
```diff
@@ -17,6 +20,14 @@ EPA is overseen by two congressional committees, the House Energy and Commerce C
+  ![Panel Discussion & Report Release October 22 6-7:30pm EST Featuring Union of Concerned Scientists, Hanah Vogel (Policy Advisor to MA Senator Ed Markey), and more](release-event.png)
 
-In these virtual workshops, we will work with community organizers, concerned residents, and university students and staff to dive into the data and stories in your legislative districts, zip codes, and counties to find out: (1) what companies are emitting; (2) what are the associated environmental health risks; (3) what is and isn’t being tracked; (4) whether local facilities are violating their ...
+  Register now!
```
Consider adding formatting here such as `<big>` and/or `<center>` to make this a clear call to action

Here's how it currently looks:
![Screen Shot 2020-10-10 at 3 30 40 PM](https://user-images.githubusercontent.com/454690/95666380-c7d46d00-0b0d-11eb-9458-0ae0377d9b1a.png)

Frijol committed:
removes web icon from bottom of page

calderEDJ committed:
Update content/home/home-content.md

Co-authored-by: Kelsey <ifoundthemeaningoflife@gmail.com>

calderEDJ committed:
changed-big-to-heading

Changed "Upcoming Events" to a heading

calderEDJ committed:
updated-upcoming-events-section

Added copy above the event graphic and reformatted "register here!" to be larger and centered

calderEDJ committed:
tabbed-register-here-type

Frijol commented on content/home/home-content.md:
```diff
@@ -17,6 +20,16 @@ EPA is overseen by two congressional committees, the House Energy and Commerce C
 We want clean air, clean water, and an end to environmental racism and injustice. 
 
-**Environmental Enforcement Watch** is series of networked online workshops that bring people together from different backgrounds to analyze the data from Environmental Protection Agency’s <a href="https://echo.epa.gov/" target=_blank >Enforcement and Compliance History Online</a> (ECHO) database. ECHO data includes reported violations of environmental laws, facility inspections, and enforcement  ...
+We believe in the public’s right to know, and that we can collaboratively design public open-source systems to analyze, discuss, and act on environmental data collectively. EPA data is available to the public through the <a href="https://echo.epa.gov/" target=_blank >Enforcement and Compliance History Online</a> (ECHO) database. ECHO data includes reported violations of environmental laws such as ...
```
This is pedantic, but semantically the links to the events page should be markdown-formatted (`[text](link url "description of link")`)

Frijol replied:
and the reports page – basically, any link that doesn't need to open in a new tab

Frijol commented:
This looks great! I have one comment on link styling, but we can merge this if you don't have time

screenshot:

![Screen Shot 2020-10-12 at 12 41 45 PM](https://user-images.githubusercontent.com/454690/95784261-5d940780-0c88-11eb-8690-4d2a03384e87.png)
